api/data_explorer/controllers/facets_controller.py

```diff
@@ -42,8 +40,13 @@ def facets_get(filter=None):  # noqa: E501
                                              es_facet._params['interval'])
                 values.append(FacetValue(name=range_str, count=count))
             else:
+                # elasticsearch-dsl returns boolean field keys as 0/1. Use the
+                # field's 'type' to convert back to boolean, if necessary.
+                if field['type'] == 'boolean':
+                    value_name = bool(value_name)
                 values.append(FacetValue(name=value_name, count=count))
```

Comment:
For the sample facets, can the facet name have "(per sample)" at the end?

(On a side note, I was wondering if we have any datasets that have more than one sample per participant. 1000 Genomes doesn't. I asked Matt about AMP PD. Eventually they will, may not be the case now.)

Reply:
Done.

I do not believe so yet, but I think its definitely a case we need to support.